src/http_seed_connection.cpp

```diff
@@ -218,7 +218,7 @@ namespace libtorrent
 
 		if (error)
 		{
-			received_bytes(0, bytes_transferred);
+			received_bytes(0, int(bytes_transferred));
```

Comment:
````c++
		virtual void sent_bytes(int bytes_payload, int bytes_protocol) = 0;
		virtual void received_bytes(int bytes_payload, int bytes_protocol) = 0;
		virtual void trancieve_ip_packet(int bytes, bool ipv6) = 0;
````
may be all of these methods should use size_t instead of int. 
there is no logic that sent or received bytes can be < 0 (special error case) ?